builtin_array.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
```

Comment:
Yeah so this also confused me too.. If I'm following the logic correctly, I believe what is happening here is that it isn't needed because it is already initialized prior to use [here](https://github.com/dop251/goja/blob/9410bcaa81d2ac8b8e7dbe7c47ba9b9375555697/builtin_array.go#L1410) in the constructor.